lib/generator.js

```diff
@@ -86,7 +86,31 @@ const wrapType = (type, customTypes = [], usedTypes) => {
   return type;
 };
 
+const splitEscaped = (line, sep = ' ') => {
+  const result = [];
+  let i = 0;
+  let start = line.indexOf('`', i);
+  while (i < line.length && start !== -1) {
+    result.push(...line.slice(i, start).split(sep));
```

Comment:
Also, this will push an empty line to the `results` if the line begins with a `` ` `` (because then start === 0 and slice will return an empty line).

Reply:
See https://github.com/metarhia/metadoc/pull/25#discussion_r270679314